static/docs/user-guide/autocomplete.md

```diff
@@ -73,6 +73,36 @@ fi
 You can `source` your `~/.bash_profile` or launch a new terminal to utilize
 completion.
 
+<details>
+
+### Click to expand if it doesn't work on Debian/Ubuntu
+
+As mentioned above, it should work out of the box. But if it does not, try these steps:
```

Comment:
does not -> doesn't ... I think we take the "less official" path with those. Am I right, @jorgeorpinel ?

Reply:
You are correct Ivan. I mean.. "You're correct" 😋 

I ty to use "does not" only in explanation about DVC commands behavior, and when special emphasis is needed.

TBH it's probably too much to expect perfect consistency with this very common term though, so either way should be OK.